uploader.json

```diff
@@ -7609,6 +7614,15 @@
                 }
             ]
         },
+        "ulimit_hard": {
+            "host": [
+                {
+                    "archive_file_name": "insights_commands/ulimit_-a_-H",
+                    "command": "/usr/bin/ulimit -a -H",
+                    "pattern": []
+                }
+            ]
+        },
```

Comment:
@jobselko please remove this change section.  This section of the json is no longer used by the current client.